### DISCUSSION TITLE
Control an Android mobile device directly from a Stream Deck +

### DISCUSSION THREAD

**Discussion by patstevenson8** [elgatosf/elgato#96] [Ideas]
I have a project that I want to use the Stream Deck + on, it involves controlling an Android app on a Samsung phone.  When looking through the development SDK and samples, it seems like this use case isn't covered.  I'd like some guidance please.  I would like to connect the Stream Deck + directly to the Android phone using a USB-C to USB-C cable.

**Comment by ChekTek** [comment]
Hi @patstevenson8

Direct Android support for Stream Deck is not on the roadmap at this time. You may have some luck communicating with Stream Deck directly via a HID protocal, but I'm not familiar with what this may look like on Android.
